scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -2040,7 +2040,8 @@ def _build_fetches(self, global_step, all_logits, losses, device_grads,
         gradient_clip = self.params.gradient_clip
         learning_rate = get_learning_rate(self.params, global_step,
                                           self.dataset.num_examples_per_epoch(),
-                                          self.model, self.batch_size)
+                                          self.model,
+                                          self.batch_size * self.num_workers)
```

Comment:
I think we should only multiply by num_workers when using Horovod. See https://github.com/tensorflow/benchmarks/issues/199#issuecomment-394437696.

Also, can you add TODO to simplify the learning rate code? Something like `# TODO(reedwm): Greatly simplify the learning rate code.`